File: Sum of All Odd Length Subarrays.py
class Solution:
    def sumOddLengthSubarrays(self, arr: List[int]) -> int:
        kq = 0
        for i in range(len(arr)):
            count, Sum = 0, 0
            for j in range(i,len(arr)):
                Sum  += arr[j]
                count +=1
                if count %2==1:
                    kq += Sum
        return kq

File: Sum of All Odd Length Subarrays.c
int sumOddLengthSubarrays(int* arr, int arrSize){

    int i,j,count =0;

    for( i =0 ; i < arrSize; i++)
    {
        for ( j = i+ 1; j < arrSize+ 1; j++ )
            if((i+j) % 2 != 0 )
            {
                int k, sum =0;
                for(k = i; k < j; k++)
                {
                    sum = sum + arr[k];
                }
                count = count + sum;

            }

    }
    return count;
}
